SwiftlineTests/SwiftlineTests/ArgsTests.swift

```diff
@@ -25,7 +25,8 @@ class ArgsTests: QuickSpec {
           "no-ff": ""]
         
         expect(Args.parsed.flags).to(equal(result))
-        expect(Args.parsed.parameters).to(equal(["Some custom one", "one", "two"]))
+        expect(Args.parsed.parameters).to(equal(["one", "two"]))
+        expect(Args.parsed.command).to(equal("Some custom one"))
```

Comment:
This expectation here is a little misleading, I think we should refactor this test.

we can set the process info above to be 
```
ProcessInfo.internalProcessInfo = DummyProcessInfo("excutable_name", "-f", "file.rb", "--integer", "1", "Some custom one", "one", "two", "--no-ff")
```

and then edit this expectation to be 
```
expect(Args.parsed.command).to(equal("excutable_name"))
```